Framework/PythonInterface/plugins/algorithms/MatchPeaks.py

```diff
@@ -236,50 +236,71 @@ def _get_peak_position(self, input_ws):
 
         # Initialisation
         peak_bin = np.ones(mtd[self._input_ws].getNumberHistograms()) * mid_bin
-        peak_plus_error = np.zeros(mtd[self._input_ws].getNumberHistograms())
 
         # Bin range: difference between mid bin and peak bin should be in this range
         tolerance = int(mid_bin / 2)
 
         x_values = mtd[self._input_ws].readX(0)
 
         for i in range(mtd[self._input_ws].getNumberHistograms()):
+
```

Comment:
Can we remove some of the blank lines? It feels like a bit too much whitespace is taken up here.